packages/web/src/components/track/desktop/TrackListItem.tsx

```diff
@@ -111,7 +111,7 @@ const TrackListItem = ({
 
   const menu: Omit<TrackMenuProps, 'children'> = {
     handle: track.user.handle,
-    includeAddToPlaylist: !track.is_stream_gated,
+    includeAddToPlaylist: true,
```

Comment:
maybe product question: should we have `addToAlbum` here?

Reply:
added